examples/binder-api.py

```diff
@@ -63,7 +63,6 @@ def build_binder(repo,
         if evt.get('phase') == 'ready':
             url = "{url}?token={token}".format(**evt)
             print("Opening %s" % url)
-            webbrowser.open(url)
```

Comment:
typo or on purpose? I quite liked this feature :-/

Reply:
whoops